### PR TITLE
Update faqs.md

### PR DESCRIPTION
**Pull request by mariapao**

### CODE REVIEW

**Comment by luisivan** [comment]
Made a couple comments, otherwise looking good. Please remember to ask @stefanobernardi for reviews from now on

File: faqs.md [hunk 1]
```diff
@@ -34,43 +34,25 @@ There are no deadlines for applications. The whole application process can take
...
-
-> Partner at Placeholder, a venture capital partnership based in New York City that invests in decentralized information networks. Prior to Placeholder, Joel led Union Square Ventures’ crypto practice and investment efforts. Before joining USV, Joel started and managed the Digital Economy Department within the Ministry of Industry and Commerce of the Dominican Republic, a government office focused on Latin American tech policy and payment system reform.
-
-[**Chris Burniske**](https://twitter.com/cburniske)
-
->Chris Burniske is a partner at Placeholder, a venture capital firm based in New York City that invests in decentralized information networks. Prior to Placeholder, he pioneered ARK Invest’s crypto efforts, leading the firm to become the first public fund manager to invest in bitcoin in 2015, and co-authored the best selling book, Cryptoassets. His commentary has been featured on national media outlets, including the Wall Street Journal, the New York Times, Fortune, and Forbes. Chris graduated Phi Beta Kappa with a BS from Stanford.
+**[Stegfano Bernardi](https://blog.aragon.org/welcoming-stefano-to-the-aragon-association/) — Executive Director. Aragon Association team**
```
Comment (0x6431346e) [inline faqs.md]:
```suggestion
**[Stefano Bernardi](https://blog.aragon.org/welcoming-stefano-to-the-aragon-association/) — Executive Director. Aragon Association team**
```

File: faqs.md [hunk 2]
```diff
@@ -34,43 +34,25 @@ There are no deadlines for applications. The whole application process can take
...
 
 ## **What will the Nest program provide to grantees besides financial backing?**
-Aragon is a very reputable name in the community. We have always stood by our values and the interests of the community. We have developed a lot of best practices of how crypto projects should be ran, and we want to help expand those as much as possible.
-
-Placeholder has years of experience in evaluating crypto projects and teams. They’ve seen many different instances of what works, and what doesn’t. They will help the projects on avoiding common mistakes and on building an open source project on a sustainable premise.
+Aragon is one of the most skilled projects in the crypto community. The development teams are formed by talented engineers that will be supporting the engineers working on the Nest projects. Also, we have developed experience, and crafted best practices, on how crypto projects should be run, and we want to help expand those as much as possible.
```
[the remark this answers is not in the document]
Comment (mariapao) [reply faqs.md]: I don't know why I couldn't see your comments before. This is why I haven't implemented the changes. I do run it through Hemingway. I imagine it is not always correct then.

File: faqs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariapao) [reply faqs.md]: This one was already implemented

**Pull request merged**